ludwig/schema/trainer.py

```diff
@@ -87,6 +72,64 @@ class TrainerConfig(schema_utils.BaseMarshmallowConfig):
         ),
     )
 
+    evaluate_training_set: bool = schema_utils.Boolean(
+        default=True, description="Whether to include the entire training set during evaluation."
+    )
+
+
+@register_trainer_schema("trainer")
+@dataclass
+class ECDTrainerConfig(BaseTrainerConfig):
+    """TrainerConfig is a dataclass that configures most of the hyperparameters used for ECD model training."""
```

Comment:
Update docstring to new name :)

Reply:
https://github.com/ludwig-ai/ludwig/pull/2027/commits/ccb524c77fec19ea6941f33528ca675cfa0f0cb2